packages/firebase_core/tizen/src/firebase_core_tizen_plugin.cc

```diff
@@ -14,9 +14,11 @@
 #include <string>
 
 #include "log.h"
-#include "messages.h"
+#include "messages.g.h"
 
-namespace {
+using ::firebase::App;
+
+namespace firebase_core_tizen {
```

Comment:
Usually flutter-tizen uses unknown namespace. (The same goes for other packages in this repo.) is there any reason to use?
https://github.com/flutter-tizen/flutterfire/pull/13/files#diff-86ddb74f044ac7ccc919e714ff91a604c2242675433d5419ce84f10d4c4b4ad4R12
It seems that there is no need to namespace in this line.